phases/bitrise_yml.go

```diff
@@ -55,16 +54,16 @@ func checkBranch(inputReader io.Reader) error {
 	return nil
 }
 
-// ParseDSLFile parses a bitrise.yml and returns a data model
-func ParseDSLFile(input io.Reader) (models.BitriseDataModel, error) {
-	var decodedDSL models.BitriseDataModel
-	if err := yaml.NewDecoder(input).Decode(&decodedDSL); err != nil {
+// ParseBitriseYMLFile parses a bitrise.yml and returns a data model
+func ParseBitriseYMLFile(input io.Reader) (models.BitriseDataModel, error) {
+	var decodedBitriseYML models.BitriseDataModel
+	if err := yaml.NewDecoder(input).Decode(&decodedBitriseYML); err != nil {
 		return models.BitriseDataModel{}, fmt.Errorf("failed to parse bitrise.yml, error: %s", err)
 	}
-	return decodedDSL, nil
+	return decodedBitriseYML, nil
 }
 
-func selectDSLFile(inputReader io.Reader) (models.BitriseDataModel, bool, error) {
+func selectBitriseYMLFile(inputReader io.Reader) (models.BitriseDataModel, bool, error) {
 	for {
 		inputPathManually, err := goinp.AskForBoolFromReaderWithDefaultValue("Input bitrise.yml path manually? (Will generate otherwise.)", false, inputReader)
```

Comment:
NOTE: i would recommend a bit more straightforward question here:

`Do you have a bitrise.yml you want to register for your new Bitrise project (if not our scanner will generate one for you)?`

Reply:
Why do you return and print as well the error couple lines below?

Reply:
The logging served as a way to see the issue with the user input. We will need to log the error at least.